### PR TITLE
Add Nepali translation for Chapter 0

### PR DESCRIPTION
**Pull request by CRLannister**
Added translations for Chapter 0 in Nepali, updated _toctree.yml, and updated github actions to include ne code.
@stevhliu

### CODE REVIEW

File: chapters/ne/chapter0/1.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+Python मा यो [*भर्चुअल इन्भाइरोमेन्ट*](https://docs.python.org/3/tutorial/venv.html) को माध्यमबाट गरिन्छ, जुन एउटा छुट्टै डाइरेक्टरी हो जसमा निश्चित Python भर्जन र एप्लिकेसनलाई चाहिने सबै प्याकेजहरू हुन्छन्। यस्तो भर्चुअल इन्भाइरोमेन्ट धेरै टूलहरूले बनाउन सक्छन्, तर हामी   
+अफिसियल Python प्याकेज [`venv`](https://docs.python.org/3/library/venv.html#module-venv) प्रयोग गर्छौं।                                                                                                                     
+
+पहिले, आफ्नो एप्लिकेसन राख्ने डाइरेक्टरी बनाउनुहोस् - उदाहरणको लागि, होम डाइरेक्टरीमा *transformers-course* नामको नयाँ फोल्डर:                                                                                                                      
+
+```
+mkdir ~/transformers-course cd ~/transformers-course
```
Comment (stevhliu) [inline chapters/ne/chapter0/1.mdx:65]: Clearer to put them on separate lines

```suggestion
mkdir ~/transformers-course
cd ~/transformers-course                                                                                                                                                               
```

File: chapters/ne/chapter0/1.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+# भर्चुअल इन्भाइरोमेन्ट सक्रिय गर्न     
+
+```                                                                                                                                                                                    
+source .env/bin/activate                                                                                                                                                                                          
+```                                                                                                                                                                                                                  
+# भर्चुअल इन्भाइरोमेन्ट निष्क्रिय गर्न  
+
+```                                                                                                                                                                                      
+deactivate                                                                                                                                                                                                        
+```
```
Comment (stevhliu) [inline chapters/ne/chapter0/1.mdx:99]: This should just be one code cell and the comments should be right before the actual command

File: chapters/ne/chapter0/1.mdx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+which python                                                                                                                                                                                                      
+```                                                                                                                                                                                                                 
+
+```out                                                                                                                                                                                                                  
+/home/<user>/transformers-course/.env/bin/python                                                                                                                                                                  
+```                                                                                                                                                                                                                   
+
+# डिपेन्डेन्सीहरू इन्स्टल गर्ने
```
Comment (stevhliu) [inline chapters/ne/chapter0/1.mdx:111]:
```suggestion
### डिपेन्डेन्सीहरू इन्स्टल गर्ने                                                                                  
```

**Review by stevhliu — COMMENTED** [review]
Thanks for your translation! Do you happen to know anyone who understands Nepali and can review your translated content? I can't read it unfortunately 😅

**Comment by CRLannister** [comment]
> Thanks for your translation! Do you happen to know anyone who understands Nepali and can review your translated content? I can't read it unfortunately 😅

Thank you for the feedback! 😊 I know someone who understands Nepali and might be able to help review this. Let me tag them here: @Bikram48. 🙌 I'll ask if they're available to take a look!

**Comment by Bikram48** [comment]
@stevhliu I have reviewed the entire translated text. Each step is clearly explained, and I can understand every word without any confusion. Is there anything else you'd like me to do?

**Review by stevhliu — APPROVED** [review]
Thanks for reviewing!

**Pull request merged**